echopype/tests/calibrate/test_cal_params.py

```diff
@@ -2,19 +2,35 @@
 
 import numpy as np
 import xarray as xr
+from xarray.testing import assert_allclose
 
 from echopype.calibrate.cal_params import (
-    CAL_PARAMS, param2da, sanitize_user_cal_dict, _get_interp_da,
-    get_cal_params_AZFP, get_cal_params_EK, get_vend_cal_params_power
+    CAL_PARAMS,
+    param2da,
+    sanitize_user_cal_dict,
+    _get_interp_da,
+    get_cal_params_AZFP,
+    get_cal_params_EK,
+    get_vend_cal_params_power,
 )
 
+DATA = np.ones((2, 200)) * 2000
+TIME_COORDINATES = np.ones(200) * 1000
 
 @pytest.fixture
-def freq_center():
+def freq_center_interp():
     return xr.DataArray(
         [[25, 55]],
         dims=["ping_time", "channel"],
-        coords={"channel": ["chA", "chB"], "ping_time": [1]}
+        coords={"channel": ["chA", "chB"], "ping_time": [1]},
```

Comment:
I notice that this `ping_time` is still only 1 value, can you update this to be like `freq_center`? Or if this is the same as `freq_center` then just update the test using this fixture to use `freq_center`? Thanks!

Reply:
Done. This was being used for `test_get_interp_da`(). I have updated the test and assertions to use `freq_center`